sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
+  "projection" : "PROJCS["NAD83 / Massachusetts Mainland", ...",
+  "bbox" : [-180,-90,180,90],
+  "footprint" : "POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))",
+  "gsd" : 0.35,
+  "file_size" : 1024,
+  "acquisition_start" : "2015-05-03T13:00:00.000",
+  "acquisition_end" : "2015-05-04T13:00:00.000",
+  "title" : "An scene from some satellite",
```

Comment:
My thought is, metadata per `dataset` where `dataset` is a URI entry in the master list. So it might be that a provider would provide multiple datasets, that could exist inside of the same bucket, but are actually different sets of imagery. The URI in the master list would point to the dataset metadata, which could have all the descriptor information like title, platform, etc. That way the master list is a URI list of datasets.